### PR TITLE
Billing scenarios after subscription 

### PR DESCRIPTION
**Pull request by vinod-demansol**
#### Summary
Billing functionalities performed after subscriptions

#### Ticket Link
https://mattermost.atlassian.net/browse/MM-37054
https://mattermost.atlassian.net/browse/MM-37051
https://mattermost.atlassian.net/browse/MM-37052

#### Release Note
```release-note
NONE
```

### CODE REVIEW

**Review by marianunez — CHANGES_REQUESTED** [review]
Thanks @vinod-demansol! Instead of testing different areas of the System Console here after purchase, could we move the tests for eaach area (Subscription screen, payment info screen) into the cypress test files for each?

File: e2e/cypress/integration/enterprise/cloud/billing/after_subscription_spec.js [hunk 1]
```diff
@@ -0,0 +1,402 @@
...
+            cy.get('.paid>span').should('have.text', 'Paid');
+
+            // # Click on see Billing history link
+            cy.contains('span', 'See Billing History').parent().click();
+
+            // * Check for billing history navigation
+            cy.url().should('include', '/billing/billing_history');
+        });
+
+        it('MM-T4134 Downloading of invoice after subscription', () => {
```
Comment (marianunez) [inline e2e/cypress/integration/enterprise/cloud/billing/after_subscription_spec.js:87]: As we discussed here: https://community-daily.mattermost.com/core/pl/x3qrge3j77n6tyqtz69zd6jyny The invoice will not show as soon as they subscribe so I believe this test would be that there is no invoice here as soon as you subscribe (you can comment the test until we get the fix for this in).

**Comment by vinod-demansol** [comment]
> Thanks @vinod-demansol! Instead of testing different areas of the System Console here after purchase, could we move the tests for eaach area (Subscription screen, payment info screen) into the cypress test files for each?

 I feel it will be difficult to control of the sequence of test cases with the way existing e2e test suite currently being executed. So I clubbed all the "after subscription" scenario's in to single file. Please let me know if i am wrong here.

**Comment by AGMETEOR** [comment]
> > > > @AGMETEOR gentle ping on this
> > > 
> > > 
> > > Going to run them and give feedback
> > 
> > 
> > > > @AGMETEOR gentle ping on this
> > > 
> > > 
> > > Going to run them and give feedback
> > 
> > 
> > Screenshot 2021-10-26 at 17 01 32
> > @vinod-demansol I am trying to run the tests locally. I am seeing some failure with `System Console - Subscription section`, see screenshot. Not sure why
> 
> @AGMETEOR , I have fixed the issue which you mentioned , Please check again and confirm.
> 
> image
> 
> Please note : last test case is failing because of existing issue (https://mattermost.atlassian.net/browse/MM-39031)

Running them again...

**Comment by AGMETEOR** [comment]
Thanks @vinod-demansol . I updated your branch with the latest master. I found this failure
Screenshot 2021-11-01 at 12 49 46

Also, the the pdf test still fails on me. I am not sure why. 
Screenshot 2021-11-01 at 12 43 43

**Comment by vinod-demansol** [comment]
@AGMETEOR , prerequisite for this script flow is trail period , could you please confirm that while executing this script , you are starting with trail period ?

**Comment by AGMETEOR** [comment]
> @AGMETEOR , prerequisite for this script flow is trail period , could you please confirm that while executing this script , you are starting with trail period ?

Yes, I started from trial period. If the tests are passing fine for you, I can approve. It could be that I am getting some of the test failures due to set up.

**Review by AGMETEOR — APPROVED** [review]

**Comment by jasonblais** [comment]
@AGMETEOR is this PR ready to merge?

cc @marianunez

**Comment by marianunez** [comment]
@saturninoabril I was trying to execute the E2E tests on this PR with the command above but doesn't seem to be kicking them off. What am I missing?

**Comment by saturninoabril** [comment]
@marianunez Not yet as there are some small items that needs to address. We'll let you and others know once ready to use. Thanks for checking :)

**Pull request merged**